symbols/arm9.yml

```diff
@@ -1883,6 +1896,15 @@ arm9:
         
         r0: dungeon id
         return: bool
+    - name: GetMaxRescueAttempts
+      address:
+        EU: 0x20516B8
+        NA: 0x2051380
+      description: |-
+        Returns the maximum rescue attempts allowed in the specified dungeon.
+        
+        r0: dungeon id
+        return: Max rescue attempts, or -1 if rescues are disabled.
```

Comment:
This can't be -1 if the return type is `uint8_t`. Do you know how the game checks this value (is it a < 0 check or a != 0xFF check)? I can't tell. If we aren't sure, I'd say to change the return type to `int8_t`.

Reply:
Oh right, the amount of rescues is signed (the game checks if it's < 0). I'll change it here and in the dungeon struct.